cognite/neat/_store/_graph_store.py

```diff
@@ -319,45 +320,43 @@ def _add_triples(
         """
         add_triples_in_batch(self.graph(named_graph), triples, batch_size)
 
-    def transform(self, transformer: Transformers, named_graph: URIRef | None = None) -> None:
+    def transform(self, transformer: Transformers, named_graph: URIRef | None = None) -> IssueList:
```

Comment:
Note that all the previous warnings are now considered errors. This is because all the conditionals leads the transformations to be not executed, thus not successful and should thus not lead to a state change.